tx-pool/src/config.rs

```diff
@@ -27,6 +27,8 @@ pub struct TxPoolConfig {
     pub max_committed_txs_hash_cache_size: usize,
     // txs with lower fee rate than this will not be relayed or be mined
     pub min_fee_rate: FeeRate,
+    // Rejects txs with scripts that might trigger known bugs
+    pub reject_known_bugs: bool,
```

Comment:
It will break existing config file because by default serde will require the field `reject_known_bugs`.  It is recommended to add `#[serde(default)]`. See https://serde.rs/field-attrs.html for details.